README.md

```diff
@@ -1 +1,69 @@
-# Awesome-book-ES6
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome-Book-ES6
+
+> A showcase of my ability as a developer is carefully displayed on this portfolio.
+
+## Built With
+
+- HTML5
+- CSS3
+- Javascript
+- markdowns
+
+## Live Demo (if available)
+
+[Live Demo Link]
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+
+### Prerequisites
+- Node Js
+
+### Setup
+- Clone this project 
+`clone link`
+
```

Comment:
- [ ] Kindly add your repository link which a user can use to clone your project.